src/cltypes.jl

```diff
@@ -20,9 +20,9 @@ for sym in enum_names(CXCursorKind)
 end
 
 CLCursor(c::CXCursor) = CXCursorMap[c.kind](c)
-Base.convert(::Type{CXCursor}, x::T) where {T<:CLCursor} = x.cursor
```

Comment:
These conversion rules seems suspicious. It's almost always wrong to define `unsafe_convert` to be the same as `convert`. Is the `convert` being used anywhere or is it just for `ccall`s?

Reply:
Just for `ccall`s, it's being called through `Base.cconvert`. Should I overload `cconvert` to `Base.cconvert(::Type{CXCursor}, x::CLCursor) = x` instead?

Reply:
Yes, you should never define `convert` only to be used for `cconvert` (That's likely a pre-0.4 legacy). I don't remember what's the default behavior for cconvert but yes, that looks like the correct `cconvert` implementation if it's not already the default behavior.